### PR TITLE
Add `background_color` to theme config

### PR DESCRIPTION
**Pull request by Eskaan**
# Description

- Added `background_color` field to the theme config on request of @VuiMuich.
- Changed some theme handling, specifically printing the all error messages when failing to run a script.

## Type of change

- [ ] Development change (no change visible to user)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation only update (no change to the factual codebase)
- [ ] This change requires a documentation update

## Updated user documentation:

Add `background_color: "#333333"` to the example in the to rename Border Colors section.

# Checklist

- [x] Ran `make test-full` locally with no errors or warnings reported
  Note: To fully reproduce CI checks, you will need to run `make test-full-nix`. Usually, this is not necessary.
- [ ] Wiki pages have been updated accordingly (to perform **after** merge)

# Todo
- Make (all) colors optional?
- Which default color?

### CODE REVIEW

**Comment by VuiMuich** [comment]
On those two TODOs:
- I welcome this suggestion, and think we might want to go all the way and make all theme fields optional then
- Should we go for a `#FFD625` like the `leftwm logo` for the default focused border color? `#333333` seems reasonable as the default background

**Comment by Eskaan** [comment]
Just a fat fail, will do that tomorrow

**Comment by VuiMuich** [comment]
Now that all colors are `Options` could you please make `border` and `margins` options as well? I think it would be just more consistent that way.

**Comment by Eskaan** [comment]
Just to clarify, not all config options are `Option`s?
I changed all colors, margins and border width are other things.

**Comment by VuiMuich** [comment]
Yes, only _all theme related_ config fields. It just feels silly, when all but two of the fields in `theme.ron` are optional, but those two are not special in any way so there would be a reason for them to be mandatory.

File: leftwm/src/config/mod.rs [hunk 1]
```diff
@@ -469,11 +469,17 @@ impl leftwm_core::Config for Config {
...
-        self.theme_setting.border_width
+        self.theme_setting.border_width.unwrap_or(1)
     }
 
     fn margin(&self) -> Margins {
-        match self.theme_setting.margin.clone().try_into() {
+        match self
+            .theme_setting
+            .margin
+            .clone()
```
Comment (VuiMuich) [inline leftwm/src/config/mod.rs:479]: Do we stell bed this clone? Might be with the `unwarp_or` we could loose this.

Comment (Eskaan) [reply leftwm/src/config/mod.rs:479]: If I remove the .clone() it does not compile. `unwrap_or(...)` moves the object.

Comment (VuiMuich) [reply leftwm/src/config/mod.rs:479]: Yeah, was just wondering why `border_width` has no clone.

Btw, can't we use `unwrap_or_default`?

Comment (Eskaan) [reply leftwm/src/config/mod.rs:479]: It's an enum, so I would have to write a manual implementation. Also all other values also use `.unwrap_or()`, example in line 531 with false.

Comment (Eskaan) [reply leftwm/src/config/mod.rs:479]: I think this is to clarify the defaults

**Pull request merged**